src/modules/CreateClassForm/components/CreateClassForm.js

```diff
@@ -49,88 +70,52 @@ function CreateClassForm({ onSubmit }) {
         <FormField
           fieldId="sponsor"
           name="sponsor"
-          options={[{ value: '1', label: 'Cardinal Solutions' }]}
-          type={Select}
+          options={sponsorOptions}
           placeholder="Select a sponsor"
+          type={Select}
         />
       </FormGroup>
 
-      <FormGroup>
-        <Label>Refreshments</Label>
-        <FormField
-          fieldId="refreshments"
-          name="refreshments"
-          options={[{ value: '1', label: 'Yes' }, { value: '2', label: 'No' }]}
-          type={Select} // TODO add a radio input
-          placeholder="Select yes or no"
-        />
-      </FormGroup>
+      <YesNoGroup id="refreshments" label="Refreshments" />
 
       <FormGroup>
         <Label>Instructor</Label>
         <FormField
           fieldId="instructor"
           name="instructor"
-          options={[{ value: '1', label: 'Andrea Jessup' }]}
-          type={Select}
+          options={instructorOptions}
           placeholder="Select the instructor"
+          type={Select}
         />
       </FormGroup>
 
-      <FormGroup>
-        <Label>Paid Event?</Label>
-        <FormField
-          fieldId="paid"
-          name="paid"
-          options={[{ value: '1', label: 'Yes' }, { value: '2', label: 'No' }]}
-          type={Select} // TODO add a radio input
-          placeholder="Select yes or no"
-        />
-      </FormGroup>
+      <YesNoGroup id="paid" label="Paid Event?" />
 
       <FormGroup>
         <Label>Class Contact</Label>
         <FormField
           fieldId="contact"
           name="contact"
-          options={[{ value: '1', label: 'Caitlin Steinert' }]}
-          type={Select} // TODO add a radio input
+          options={contactOptions}
```

Comment:
This one shows up with 'null' as its last option in the dropdown. Wondering if we need to be stripping off non-values somewhere?